pyproject.toml

```diff
@@ -37,6 +37,7 @@ opentelemetry-instrumentation-httpx = "^0.45b0"
 pillow = "10.3.0"
 azure-mgmt-cognitiveservices = "^13.5.0"
 jsonschema = "^4.21.1"
+semantic-kernel = {version = "^0.9.6b1", python = "<3.13"}
```

Comment:
There's also a semantic kernel vscode extension (haven't had a play) - so interested in what it gives us. Feel free to resolve without any action!

Reply:
Unsure tbh, I tried using it when I was following tutorials but the code it generated was out of date. It hasn't been since October